src/Plugin/views/field/RetryOperation.php

```diff
@@ -47,7 +47,7 @@ public function query() {
    * {@inheritdoc}
    */
   public function access(AccountInterface $account) {
-    return $account->hasPermission('administer advancedqueue');
+    return TRUE;
```

Comment:
Shouldn't this be something like “can do stuff with webforms or submissions”?

Reply:
It's really no different from displaying the the message or jobtype in the list. They don't have an access check.
To perform the actual action you still need to pass route action check

Reply:
I suppose it could be "access webform submission log" since that is the requirement for the route action

Reply:
Changed